fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -6385,7 +6385,10 @@ public void replayDropFunction(FunctionSearchDesc functionSearchDesc) {
 
     public void setConfig(AdminSetConfigStmt stmt) throws DdlException {
         Map<String, String> configs = stmt.getConfigs();
-        Preconditions.checkState(configs.size() == 1);
+        // configs can not be null
+        if (configs.size() != 1) {
```

Comment:
This should be done in `AdminSetConfigStmt`'s `analyze` method.

Reply:
done